internal/http/services/owncloud/ocs/handlers/apps/sharing/sharees/sharees.go

```diff
@@ -132,3 +143,11 @@ func (h *Handler) groupAsMatch(g *grouppb.Group) *conversions.MatchData {
 func (h *Handler) getAdditionalInfoAttribute(u *userpb.User) string {
 	return templates.WithUser(u, h.additionalInfoAttribute)
 }
+
+func (h *Handler) isExactUser(match *conversions.MatchData, term string) bool {
```

Comment:
```suggestion
func (h *Handler) isExactMatch(match *conversions.MatchData, term string) bool {
```
Not user specific, so naming is a bit off. But not critical

Reply:
Already changed.